src/components/document/document.tsx

```diff
@@ -89,11 +89,13 @@ const CopyButton = ({ onClick }: { onClick: () => void }) => {
   );
 };
 
-const DeleteButton = ({ onClick }: { onClick: () => void }) => {
-  return (
-    <IconButton icon="delete" key="delete" className="action icon-delete"
-                onClickButton={onClick} />
-  );
+const DeleteButton = ({ onClick, enabled }: { onClick: () => void, enabled: boolean }) => {
+    const visibility = enabled ? "enabled" : "disabled";
```

Comment:
```suggestion
    const enabledClass = enabled ? "enabled" : "disabled";
```
"visibility" isn't the right word here. There may be a better name than the one I proposed, but it's closer.

Reply:
Would `toggleDisabledButton` or `toggleButton` be a possible option here?